src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -384,6 +385,7 @@ public void onTraversalStart() {
             default:
                 throw new GATKException("Unsupported output format type specified: " + outputFormatType.toString());
         }
+        logger.info("Creating a " + outputFormatType.toString() + " file for output: " + outputFile.toURI().toString());
```

Comment:
unnecessary `.toString()` calls, as arguments to the String concatenation operator are automatically converted to string

Reply:
Fixed.